utils/date.js

```diff
@@ -3,7 +3,12 @@ export function minDate() {
   const dd = `${date.getDate()}`.padStart(2, '0');
   const mm = `${date.getMonth() + 1}`.padStart(2, '0');
   const yyyy = date.getFullYear();
-  return `${yyyy}-${mm}-${dd}`;
+  const hours = date.getHours();
+  const min = date.getMinutes();
+  const sec = '00';
+  const mill = '00';
+
+  return `${yyyy}-${mm}-${dd}T${hours}:${min}:${sec}.${mill}`;
```

Comment:
why don't you use the moment library for this?

Reply:
i changed it, thanks